### DISCUSSION TITLE
Does quartz support clustering in multi tenant databases?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2223] [Q&A]
Hi I have a multi tenant .net 6 web api which is using quartz.net as scheduler. Each tenant has its own database and so each database has its own quartz tables as well and their own jobs to be scheduled.

I want to run multiple instances of this service. Load wise it should be able to handle 10k jobs at a given time 

so what is the best practices for such implementation.

is there any sample project with such implementation, please share.

FYI @lahma

**Comment by lahma** [comment]
You should be all set if you have scheduler per database (tenant) in this scenario.
